### ISSUE TITLE
Package Missing

### ISSUE THREAD

**Issue by Griffin97** [johannestang/forecastexp#1]
Greetings ,
The link to installation of this package (install_github('johannestang/forecastexp') is not working any solution to a package that replicates the results.

The following error is incurred upon installation on Mac book pro.

> install_github('johannestang/forecastexp')
Downloading GitHub repo johannestang/forecastexp@HEAD
These packages have more recent versions available.
It is recommended to update all of them.
Which would you like to update?

1: All                           
2: CRAN packages only            
3: None                          
4: Rcpp   (1.0.5 -> 1.0.6) [CRAN]
5: glmnet (4.0-2 -> 4.1  ) [CRAN]

Enter one or more numbers, or an empty line to skip updates: 1
Rcpp   (1.0.5 -> 1.0.6) [CRAN]
glmnet (4.0-2 -> 4.1  ) [CRAN]
lars   (NA    -> 1.2  ) [CRAN]
Installing 3 packages: Rcpp, glmnet, lars
Error: Failed to install 'forecastexp' from GitHub:
  (converted from warning) unable to access index for repository http://www.omegahat.net/R/bin/macosx/contrib/4.0:
  cannot open URL 'http://www.omegahat.net/R/bin/macosx/contrib/4.0/PACKAGES'

**Comment by johannestang** [comment]
I don't believe the problem you are facing is related to this package. I just tried installing it on both versions 3 and 4 of R and it worked just fine. I am, however, using Linux and not macOS. I would suggest that you try switching to a different CRAN repository or that you try installing the dependencies manually instead of having `install_github` do it for you.
